Neos.Flow/Documentation/Quickstart/index.rst

```diff
@@ -265,10 +261,10 @@ action method like this one::
     /**
      * This action outputs a custom greeting
      *
-     * @param string $name your name
```

Comment:
Does that actually work in all cases? Flow used to only look at the docblock, does it parse the type declaration now as well? If I did miss something there, all the better! 🎉

Reply:
Yes, that works. I think it does since 2016 actually (#394) :)

Reply:
🤯👍